docs/pages/programming/jdbc_api.md

```diff
@@ -38,7 +40,23 @@ The JAR files normally can be found in `~/.m2/repository/com/bloomberg/comdb2/cd
 Another option is to build the driver inside a Docker container by running `make jdbc-docker-build` in `cdb2jdbc` (JAR files will be written
 to `cdb2jdbc/maven.m2/repository/com/bloomberg/comdb2/cdb2jdbc/2.0.0/`)
 
+#### Gradle
+
+Another way to install cdb2jdbc from source is the gradle wrapper. The following software packages are required:
+
+*   JDK 1.7 or above
+
+> Note that existing installations of gradle and protocol buffers **are not** required for the gradle build
+
+Once you check out the source and have all the required software installed on the system, change directory to cdb2jdbc under comdb2 source and type `./gradlew install` or `./gradlew.bat install` for Windows.
+
+```shell
+cd cdb2jdbc
+./gradlew install
+```
 
+cdb2jdbc should be successfully installed in your local Maven repository.
```

Comment:
It might be worth mentioning that the `install` task is actually a Maven phase and the maven-gradle plugin is used to build the driver under Maven.